src/evict/evict_lru.c

```diff
@@ -1569,7 +1569,8 @@ __evict_walk(WT_SESSION_IMPL *session, WT_EVICT_QUEUE *queue)
         /*
          * If we are filling the queue, skip files that haven't been useful in the past.
          */
-        if (btree->evict_walk_period != 0 && btree->evict_walk_skips++ < btree->evict_walk_period) {
+        if (__wt_atomic_load32(&btree->evict_walk_period) != 0 &&
```

Comment:
Should this be pulled into a local variable?

Reply:
Updated in [c088090](https://github.com/wiredtiger/wiredtiger/pull/10635/commits/c08809048d3122fa26da500e7ecda927e373e41d)